crates/bevy_ui/src/node_bundles.rs

```diff
@@ -339,14 +374,14 @@ pub struct ButtonBundle {
 impl Default for ButtonBundle {
     fn default() -> Self {
         Self {
-            focus_policy: FocusPolicy::Block,
             node: Default::default(),
             button: Default::default(),
             style: Default::default(),
-            border_color: BorderColor(LegacyColor::NONE),
             interaction: Default::default(),
+            focus_policy: FocusPolicy::Block,
             background_color: Default::default(),
-            image: Default::default(),
+            border_color: BorderColor(LegacyColor::NONE),
+            image: UiImage::default().with_color(LegacyColor::NONE),
```

Comment:
This behaviour is inconsisten with `ImageBundle` and `AtlasImageBundle` which have a `BackgroundColor` defaulting to `LegacyColor::None`, which is probably a better default, or at least one that keeps the current behaviour.

In the continuity of previous comments, we could also just remove the component from the bundle, and it would keep iso-functionality